tests/test_browser_model.py

```diff
@@ -36,7 +36,15 @@ def test_close_window(foxpuppet):
 def test_switch_to(foxpuppet, selenium):
     """Test Switch to function"""
     foxpuppet.browser.open_window()
-
     # Switch to originally window opened by pytest
     foxpuppet.browser.switch_to()
     assert foxpuppet.browser.handle == selenium.current_window_handle
+
+
+def test_tracking_protection_shield(foxpuppet, selenium):
+    """Tests if the tracking protection icon displays"""
+    browser = foxpuppet.browser.open_window(private=True)
+    browser.switch_to()
+    assert not browser.navbar.is_tracking_shield_displayed
+    selenium.get('http://cnn.com')
```

Comment:
Ups, something scrambled my comment yesterday. so what I wanted to say is that this looks to be too complex to do right now. Maybe we should go ahead with a remote site. Maybe use one which is less annoying than CNN.

We just got a new mixed content setup (bug 1351694) and it looks like we could use one of those pages.